index.html

```diff
@@ -4,16 +4,14 @@
         <link type="text/css" rel="stylesheet" href="index.css">
     </head>
     <body>    
-          <ul class="Step">
-            <li class="Step__active">
-              <img src="image/Ellipse-1.png" class="img__ellipse">
-              <img src="image/Path-1.png" class="img__path">
-              <p class="Step__text">Step 1</p>
+          <ul class="steps">
+            <li class="steps__item steps__item--enabled">
+              <img src="image/active.png" class="step__image">
+              <p class="step__text">Step 1</p>
             </li>
-            <li >
-                <img src="image/Ellipse-2.png" class="img__ellipse">
-                <img src="image/Path-2.png" class="img__path">
-                <p class="Step__text">Step 2</p>
+            <li class="steps__item steps__item--disenabled">
```

Comment:
Please correct the class name
```suggestion
            <li class="steps__item steps__item--disabled">
```